### ISSUE TITLE
Helm Repo documentation and/or hosting the chart

### ISSUE THREAD

**Issue by am-rpa** [digihunch/korthweb#7]
First, I'd like to say I'm ecstatic to see this project as I think this type of work will help bring Orthanc in view of many other enterprises. 

One thing I think would help provide clarity to users that are used to running "helm repo add" commands would be to document the steps of cloning the repo and then setting changing the directory to it before running the "helm dependency update" command. 

The other alternative that I think would be better would be to use GitHub Pages to create a public helm repo. [Here ](https://medium.com/@mattiaperi/create-a-public-helm-chart-repository-with-github-pages-49b180dbb417) is an article I've used in the past to do this if there is interest. 

If the documentation route is preferred, I'd be happy to submit a PR detailing those steps.

**Comment by digihunch** [comment]
Hi @am-rpa  Thanks for the comment. For background, my original approach is to build my own Helm Chart to reference dependency resources such as postgres-ha as child chart. 
As I decided to add Istio, I ran into a number of issues when trying to expand the original Helm chart, which led to a rethinking of the role of Helm in this project.
Going forward, two deployment approaches will be included:
- Helm package (as-is, and to your point, the git clone step will be explicitly spelled out) for installation without Istio.
- GitOps approach using FlexCD (currently a [WIP](https://github.com/digihunch/korthops)) that installs istio as well.
Any further comments, expectations will be appreciated.
